src/components/EventCard.tsx

```diff
@@ -0,0 +1,61 @@
+import styles from "../css/EventCard.module.css";
+
+const EventCard = ({ title, startDateTime, location }) => {
+	return (
+		<div className={`${styles["event-card"]}`}>
+			<div className={`${styles["left-container"]}`}>
+				<div className={`${styles["date-container"]}`}>
+					<h3>Date</h3>
+				</div>
+			</div>
+			<div className={`${styles["right-container"]}`}>
+				<h2>{title}</h2>
+				<div className={`${styles["meeting-time-container"]}`}>
+					<div className={`${styles["icon-container"]}`}>
+						<svg
+							xmlns="http://www.w3.org/2000/svg"
+							fill="none"
+							viewBox="0 0 24 24"
+							strokeWidth={1.5}
+							stroke="currentColor"
+							className={`${styles["meeting-time-icon"]}`}
+						>
+							<path
+								strokeLinecap="round"
+								strokeLinejoin="round"
+								d="M12 6v6h4.5m4.5 0a9 9 0 1 1-18 0 9 9 0 0 1 18 0Z"
+							/>
+						</svg>
```

Comment:
We may want to include the heroicons library instead of referencing the SVG. Documentation on how to do that is located here: https://github.com/tailwindlabs/heroicons?tab=readme-ov-file#react